.github/workflows/license.yml

```diff
@@ -0,0 +1,19 @@
+name: Update copyright year(s) in license file
+
+on:
+  schedule:
+    - cron: "0 3 1 1 *"
+  workflow_dispatch:
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - uses: FantasticFiasco/action-update-license-year@v2
+      with:
+        path: LICENSE.md
+        labels: 'cherry-pick:release/oss-v5,cherry-pick:release/oss-v20.10,cherry-pick:release/oss-v21.10'
```

Comment:
**SUGGESTION:** It might be worth putting information in README or Trainstation about the need to add more labels here when the new LTS release is released. The other option is to extend the cherry-pick bot to have labels like "last-lts" "current-lts" that will select the proper branch by convention. This suggestion is non-blocking. It could be enhanced later.